plugins/org.eclipse.fordiac.ide.debug/src/org/eclipse/fordiac/ide/debug/fb/FBLaunchEventQueue.java

```diff
@@ -53,6 +72,7 @@ public Event receiveInputEvent() throws InterruptedException {
 			incrementEventCount(result);
 			if (repeat.get()) {
 				queue.add(result);
+				setDebugTime();
```

Comment:
It seems this does not properly set the time for the first event when repeat is off.

Reply:
done